src/routes/safe/store/actions/addNewTransactions.ts

```diff
@@ -0,0 +1,5 @@
+import { createAction } from 'redux-actions'
+
+export const ADD_NEW_TRANSACTIONS = 'ADD_NEW_TRANSACTIONS'
+
+export const addNewTransactions = createAction(ADD_NEW_TRANSACTIONS)
```

Comment:
can we rename it? I think the name is confusing, we are not adding TXs we are fetching them. perhaps `getTransactions` or `fetchTransactions`.

Reply:
We are adding new txs, I think the name is ok, it's the same logic that we have for example now between `fetchSafe` and `addSafe`